frameworks/PHP/symfony/src/Controller/DbController.php

```diff
@@ -41,10 +43,10 @@ public function queries(Request $request): JsonResponse
 
         // possibility for enhancement is the use of SplFixedArray -> http://php.net/manual/de/class.splfixedarray.php
         $worlds = [];
-
-        for ($i = 0; $i < $queries; ++$i) {
-            $worlds[] = $this->worldRepository->find(mt_rand(1, 10000));
+        foreach ($this->getUniqueRandomNumbers($queries) as $id) {
+            $worlds[] = $this->worldRepository->find($id);
         }
+        $this->entityManager->clear();
```

Comment:
I'll add a Swoole Handler to perform this only for swoole env

Reply:
You just need to make sure that this operation is performed before sending the Http response.

Reply:
this is a clear (remove internal cache). not a flush